components/NavBar.tsx

```diff
@@ -2,36 +2,42 @@ import React from 'react';
 import Image from 'next/image';
 import stytchLogo from '/public/stytch-logo.svg';
 import nextjsLogo from '/public/nextjs-logotype-dark.svg';
+import Link from 'next/link';
 
 function NavBar() {
   return (
     <div style={styles.container}>
       <div style={styles.logoContainer}>
-        <a href="https://stytch.com" rel="noopener noreferrer" target="_blank">
+        <Link href={'/'}>
```

Comment:
imo I'd either of this go to stytch.com or make it go to the home of this app

Reply:
I personally have clicked on the logo many times thinking it would take me back to the home page. Going to `/` takes you to the home page

Reply:
Ah, this is completely my mistake. Agreed with this change.